src/util/cilfacade.ml

```diff
@@ -17,7 +17,7 @@ let init () =
   print_CIL_Input := true
 
 let currentStatement = ref dummyStmt
```

Comment:
Since we touched this anyway, maybe also make `current_statement`?
Then naming is consistent at least locally here